pkg/codegen/schema/loader.go

```diff
@@ -149,6 +149,17 @@ func (f getSchemaNotImplemented) Error() string {
 	return fmt.Sprintf("it looks like GetSchema is not implemented")
 }
 
```

Comment:
We can move the regex outside the function:
```suggestion
var schemaIsEmptyRE = regexp.MustCompile(`\s*\{\s*\}\s*$`)
```
and then `schemaIsEmpty` becomes `return schemaIsEmptyRE.matches(schemaBytes)`. This will clean up the code and ensure that the regex is valid.

Reply:
Good change - implemented and will merge with that.